guides/common/modules/proc_adding-vmware-details-to-a-compute-profile.adoc

```diff
@@ -3,37 +3,45 @@
 
 You can predefine certain hardware settings for virtual machines on VMware vSphere.
 You achieve this through adding these hardware settings to a compute profile.
-To use the CLI instead of the {ProjectWebUI}, see the xref:cli-adding-vmware-vsphere-details-to-a-compute-profile_{context}[].
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-Adding_VMware_Details_to_a_Compute_Profile_{context}[].
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Infrastructure* > *Compute Profiles* and, in the Compute Profiles window, click the name of the compute profile, and then click the vSphere connection.
-. In the *CPUs* field, enter the number of CPUs to allocate to the new host.
+. In the {ProjectWebUI}, navigate to *Infrastructure* > *Compute Profiles*.
+. Select a compute profile.
+. Select a Vmware compute resource.
+. In the *CPUs* field, enter the number of CPUs to allocate to the host.
 . In the *Cores per socket* field, enter the number of cores to allocate to each CPU.
-. In the *Memory* field, enter the amount of memory to allocate to the new host.
-. In the *Cluster* field, enter the name of the target host cluster on the VMware environment.
+. In the *Memory* field, enter the amount of memory in MiB to allocate to the host.
+. In the *Firmware* checkbox, select either _BIOS_ or _UEFI_ as firmware for the host.
+By default, this is set to _automatic_.
+. In the *Cluster* list, select the name of the target host cluster on the VMware environment.
 . From the *Resource pool* list, select an available resource allocations for the host.
-. In the *Folder* field, enter the folder to organize the host.
+. In the *Folder* list, select the folder to organize the host.
 . From the *Guest OS* list, select the operating system you want to use in VMware vSphere.
-. From the *SCSI controller* list, select the disk access method for the host.
 . From the *Virtual H/W version* list, select the underlying VMware hardware abstraction to use for virtual machines.
-. You can select the *Memory hot add* or *CPU hot add* check boxes if you want to add more resources while the virtual machine is powered.
-. From the *Image* list, select the image to use if performing image-based provisioning.
+. If you want to add more memory while the virtual machine is powered on, select the *Memory hot add* check box.
```

Comment:
As per the IBM styleguide, this should be used as a single word. Please see page 305 ibm-style-documentation.

Reply:
Do you mean "check box"? My version of IBM style guide explicitely says "two words":

<img width="247" alt="Screenshot 2022-07-27 at 11 25 33" src="https://user-images.githubusercontent.com/12595287/181212646-7a9bd99d-3a8a-48f9-b37d-453c1a55d260.png">


```
$ git grep "check box" | grep -v "drafts/doc-10-steps/" | wc
     120    2199   21555
$ git grep "checkbox"  | grep -v "drafts/doc-10-steps/" | wc
      12     189    1915
```

Reply:
Nope, I mean checkbox, as a single word.

Reply:
OK, but IBM style guide says it's _two_ words. Does your version say it's one word?

Reply:
Yes!

Reply:
Thanks for letting me know! Sadly IBM style guide is not open source/freely available ... 👀